README.md

```diff
@@ -4,161 +4,12 @@
-  `BUILDKITE_API_TOKEN` environment variable.
-- `organization` - (Required) This is the Buildkite organization slug. It must be provided, but can also be sourced from
-  the `BUILDKITE_ORGANIZATION` environment variable. The token requires GraphQL access and the `write_pipelines` scope.  
-
-### `buildkite_agent_token` resource
-
-This resource allows you to create and manage agent tokens.
-
-#### Example
-
-```terraform
-resource "buildkite_agent_token" "fleet" {
-    description = "token used by build fleet"
-}
-```
-
-#### Argument reference
-
-- `description` - (Optional) This is the description of the agent token.
-
-#### Attribute reference
-
-- `token` - The value of the created agent token.
-- `uuid` - The UUID of the token.
-
-### `buildkite_pipeline` resource
-
-This resource allows you to create pipelines for repositories.
-
-#### Example
-
-```terraform
-# in ./steps.yml:
-# steps:
-#   - label: ':pipeline:'
-#     command: buildkite-agent upload
-        
-resource "buildkite_pipeline" "repo2" {
-    name = "repo2"
-    repository = "git@github.com:org/repo2"
-    steps = file("./steps.yml")
-
-    team {
-        slug = "everyone"
-        access_level = "READ_ONLY"
-    }
-}
-```
-
-#### Argument reference
-
-- `name` - (Required) The name of the pipeline.
-- `description` - (Optional) A description of the pipeline.
-- `repository` - (Required) The git URL of the repository.
-- `default_branch` - (Optional) The default branch to prefill when new builds are created or triggered.
-- `branch_configuration` - (Optional) Limit which branches and tags cause new builds to be created, either via a code push or via the Builds REST API.
-- `steps` - (Required) The string YAML steps to run the pipeline.
-- `team` - (Optional) Set team access for the pipeline. Can be specified multiple times for each team.  
-
-The `team` block supports:
-
-- `slug` - (Required) The buildkite slug of the team.
-- `access_level` - (Required) The level of access to grant. Must be one of `READ_ONLY`, `BUILD_AND_READ` or `MANAGE_BUILD_AND_READ`.
-
-#### Attribute reference
-
-- `webhook_url` - The Buildkite webhook URL to configure on the repository to trigger builds on this pipeline.
-- `slug` - The slug of the created pipeline.
-
-### Importing existing resources
-
-The following resources support importing:
-
-- `buildkite_agent_token` using its GraphQL ID (not UUID)
-    - eg: `terraform import buildkite_agent_token.fleet QWdlbnRUb2tlbi0tLTQzNWNhZDU4LWU4MWQtNDVhZi04NjM3LWIxY2Y4MDcwMjM4ZA==`
-- `buildkite_pipeline` using its GraphQL ID (not UUID)
-    - eg: `terraform import buildkite_pipeline.fleet UGlwZWxpbmUtLS00MzVjYWQ1OC1lODFkLTQ1YWYtODYzNy1iMWNmODA3MDIzOGQ=`
-
-## Local development
-
-Contributions welcome!!
+- [Using the provider](https://registry.terraform.io/providers/jradtilbrook/buildkite/latest/docs)
+- [Provider Developement](.github/DEVELOPMENT.md)
```

Comment:
Lets leave the installation and local development parts here until it's confirmed hosted and working and rendering properly on the registry.